src/reporting/ReportingWindow.sol

```diff
@@ -289,4 +291,22 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable {
         numberOfReportsByMarket[_market] += 1;
         return true;
     }
+
+    function getWinningPayoutDistributionHashFromFork(Market _market) constant returns (bytes32) {
```

Comment:
If that is what it comes to for reducing market contract size, I recommend we just create a `MarketExtensions` (name TBD) function.  That way it is at least clear, and users can do `using MarketExtensions for Market;` in their contract to make it _feel_ like its all one contract.